recipe/LICENSE

```diff
@@ -0,0 +1,202 @@
+                                 Apache License
```

Comment:
@dougalsutherland 
should we ask upstream to package the license in the .whl file?

Reply:
@dougalsutherland is already on top of it with [PR #151](https://github.com/peterbrittain/asciimatics/pull/151)